language_specific/java/running_examples.rst

```diff
@@ -0,0 +1,9 @@
+Since third-party Java libraries generally don't have a standard place to install them, you will likely have to set the ``CLASSPATH`` variable appropriately to indicate that you wish to use the Xapian Java bindings.
+
+There are two parts to the bindings: a jarfile (``xapian.jar``) containing the Java classes, and the JNI library (``libxapian_jni.jnilib``) that connects them to Xapian itself. The easiest way to get this working is to copy those two files to the top-level directory of this repository. If you built your own Java bindings, the files will be in ``java/built`` in the bindings source code. Then you can use the following classpath::
+
+  xapian.jar:libxapian_jni.jnilib:.
```

Comment:
`.jnilib` is an OS-X-ism.  The JNI library extension varies by platform but it's `.so` on Linux for example.  It doesn't seem to really be possible to bundle the JNI library into the `.jar` (which isn't too surprising, as the OS will want to memory map it).

I've seen solutions which extract the JNI library to a temporary path but that seems a bad idea (it's hard to ensure it gets cleaned up in every situation, and the temporary directory could be mounted `noexec` for security reasons).